### PR TITLE
CCIBI-3436 UX - Cast - Pagination Enhancements

### PR DESCRIPTION
**Pull request by andrey-liventsev-ontarget**
Pagination enhancements with breaking changes:
- Pagination property `btnSize` was removed
- Pagination properties `previousText`/`nextText` have been replaced with `PageButtonFirstLastComponent` property
- `rowsText` default value was removed

### CODE REVIEW

**Comment by johnny-five** [comment]
I, @cachapman, have checked:

### Style/Standards/Process

* [ ] If there is commented out code that should be deleted
* [ ] That the change implements what was requested in the [ticket](https://technekes.atlassian.net/browse/CCIBI-3436)
* [ ] For adequate test coverage for changed lines or critical code paths

### Code Quality

* [ ] Do the classes follow [CodeClimate 7 Patterns to Refactor](http://blog.codeclimate.com/blog/2012/10/17/7-ways-to-decompose-fat-activerecord-models/)
* [ ] If there are more idiomatic ways of using our frameworks/libraries
* [ ] Are there alternative implementations or refactors that increase readability/understandability and/or maintainability
* [ ] I can actually understand what the code is doing (if you're struggling to understand, then others may as well)
* [ ] This change is ready for production, if not should there should be a [feature flag](https://github.com/voormedia/flipflop)

### Security and Privacy

* [ ] Do the changes raise any security or privacy concerns

File: src/Table/Table.component.tsx [hunk 1]
```diff
@@ -268,6 +266,9 @@ export class Table extends React.Component<Props> {
       theme,
       ...props
     } = this.props;
+
+    console.log('rowsText = ', this.props.rowsText);
```
Comment (cachapman) [inline src/Table/Table.component.tsx:270]: Normal to have console.log in here?

Comment (andrey-liventsev-ontarget) [reply src/Table/Table.component.tsx:270]: @cachapman I used to trust eslint checks but it won't work for cast-ui

Comment (cachapman) [reply src/Table/Table.component.tsx]: No worries, hence why we have peer reviews :-)

**Comment by ben-dalton** [comment]
This looks good to me.

**Pull request merged**